engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/serialization/KafkaSerializationSchemaFactory.scala

```diff
@@ -1,48 +1,41 @@
 package pl.touk.nussknacker.engine.kafka.serialization
 
-
-import java.nio.charset.StandardCharsets
-import java.{lang, util}
-import java.util.UUID
-
 import org.apache.flink.streaming.connectors.kafka.KafkaSerializationSchema
 import org.apache.kafka.clients.producer.ProducerRecord
 import org.apache.kafka.common.serialization.Serializer
 import pl.touk.nussknacker.engine.kafka.KafkaConfig
+import java.lang
 
 /**
   * Factory class for Flink's KeyedSerializationSchema. It is extracted for  purpose when for creation
   * of KeyedSerializationSchema are needed additional information like list of topics and configuration.
   *
   * @tparam T type of serialized object
   */
-trait SerializationSchemaFactory[T] extends Serializable {
-
+trait KafkaSerializationSchemaFactory[T] extends Serializable {
```

Comment:
IMO we don't need this abstraction. If someone has simple case, he/she only need `KafkaSerializationSchema`. You have concrete implementations only in avro module. So it looks like it was only for avro purpose?

Reply:
Ok, I see now, that it was before and is important, because `ProducerRecord` needs topic.

Reply:
we explained it